css/index.css

```diff
@@ -3,144 +3,402 @@
+}
+
+.logo {
+  width: 10%;
+}
+
+header nav a {
+  color: black;
+  text-decoration: none;
+}
+
+.headimg {
+  background-image: url("/img/jumbo.jpg");
+  height: 10vh;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: top center;
 }
 
+.head {
+  background-image: url("../img/jumbo-about.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: top center;
+  height: 10vh;
+}
 .container {
-    width: 800px;
-    margin: 0 auto;
+  width: 800px;
+  margin: 0 auto;
+}
+.ab {
+  margin-top: 3%;
+  margin-bottom: 3%;
+}
+
+.bottom {
+  margin-top: 3%;
+  margin-bottom: 3%;
 }
 
 .top-content {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-bottom: 20px;
+  border-bottom: 1px dashed black;
 }
 
 .top-content .text-container {
-    width: 48%;
-    padding: 0 1%;
-    padding-bottom: 20px;  
+  width: 48%;
+  padding: 0 1%;
+  padding-bottom: 20px;
 }
 
 .middle-content {
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+  margin-bottom: 20px;
+  border-bottom: 1px dashed black;
 }
 
 .middle-content h2 {
-    padding: 0 2%;
-    margin-bottom: 0;
+  padding: 0 0%;
+  margin-bottom: 0;
 }
 
 .middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
 }
 
 .middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
-    color: white;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+  width: 12.5%;
+  height: 100px;
+  margin: 20px 2.5%;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.bottom-content,
+footer {
+  display: flex;
+  /* margin: 0 2% 20px; */
+  justify-content: space-around;
+  width: 800px;
+  margin: 0 auto;
 }
 
 .bottom-content {
-    display: flex;
-    margin: 0 2% 20px;
-    justify-content: space-around;
+  padding-bottom: 3%;
+  align-items: left;
 }
 
 .bottom-content .text-container {
-    padding-right: 4%;
+  padding-right: 4%;
 }
 
 .bottom-content .text-container:last-child {
-    padding-right: 0;
+  padding-right: 0;
 }
 
 footer {
-    width: 100%;
-    background: black;
+  width: 800px;
+  background: black;
 }
 
 footer nav {
-    width: 60%;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+  width: 60%;
+  display: flex;
+  justify-content: space-between;
+  align-items: left;
+  padding: 20px 2%;
+  font-size: 14px;
+  margin-right: 30%;
 }
 
 footer nav a {
-    color: white;
-    text-decoration: none;
-}
\ No newline at end of file
+  color: white;
+  text-decoration: none;
+  text-align: left;
+}
+
+/* My edits */
+
+/* Box Colors */
+
```

Comment:
Nice work overriding only the color of each individual box and letting the parent container do the rest of the styling.